rosbridge_library/src/rosbridge_library/capabilities/action_client.py

```diff
@@ -0,0 +1,184 @@
+import fnmatch
+from functools import partial
+
+from rosbridge_library.capability import Capability
+from rosbridge_library.internal.actions import ActionClientHandle, GoalHandle
+from rosbridge_library.internal.message_conversion import extract_values
+
+
+class ActionClientRequests(Capability):
```

Comment:
This doesn't seem like it's named consistently as a "capability" compared to the others.

I think maybe this should be called `SendActionGoal` or something?